CMakeLists.txt

```diff
@@ -102,6 +104,29 @@ else(MSVC)
   else()
     set(SUPPORT_MSSE2 FALSE)
   endif()
+  # For cross complication, turn off flag if target device does not support it
+  if(USE_F16C AND NOT MSVC)
```

Comment:
It is already guaranteed that MSVC is not present at this point

Reply:
Yeah corrected it thanks